lib/helpers/answerLookup.js

```diff
@@ -0,0 +1,27 @@
+function lookupName(index) {
+    const answerList = {
+        true: 'Yes',
+        false: 'No',
+        once: 'Once',
+        'over-a-period-of-time': 'Over a period of time',
+        'i-was-underage': 'I was under 18',
```

Comment:
did Julie say this was incorrect? doesn't `i-was-underage` appear somewhere when i should be `i-was-under-18`?

Reply:
Yes, here. Previously this id became "I was underage". Now it will appear as "I was under 18"